docker-app/qfieldcloud/core/templates/admin/project_files_widget.html

```diff
@@ -68,7 +68,7 @@
         overflow-y: auto;
     }
 
-    .qfc-admin-projects-files-actions-reload-btn,
+    /*.qfc-admin-projects-files-actions-reload-btn,
```

Comment:
Don't leave commented stuff without a comment why you commented it. If it is no longer required, just delete it, git will keep a copy of the old code if someone needs it.